plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/StmtConversionVisitor.kt

```diff
@@ -164,13 +187,18 @@ class StmtConversionVisitor : FirVisitor<Exp, StmtConversionContext>() {
             return specialFunc.convertCall(getArgs(), data)
         }
 
-        val symbol = functionCall.calleeReference.resolved!!.resolvedSymbol as FirNamedFunctionSymbol
-        val calleeSig = data.add(symbol)
+        val calleeSig = when (val symbol = functionCall.calleeReference.resolved!!.resolvedSymbol) {
```

Comment:
Nit: I would move the `val symbol = ...` expression to the line above before the when